Krivorotova Tatyana/style.css

```diff
@@ -0,0 +1,101 @@
+body{
+    margin: 0;
+}
+
+#widget{
```

Comment:
It's not good idea to use id selector for styling your markup. This selector has big specifity and it will be hard to override it if needed (see https://developer.mozilla.org/ru/docs/Web/CSS/Specificity). In general, you should use classes for this purpose.